hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -86,23 +84,23 @@
 
     private static final Random RANDOM_PARTITION_ID_GENERATOR = new Random();
```

Comment:
Even though this isn't your change, a Random officially is not thread-safe. So it depends on the platform if this can cause problems or not and relying on it is asking for problems.